newsroom/agenda/email.py

```diff
@@ -83,6 +83,9 @@ def send_coverage_request_email(user, message, item):
     name = "{} {}".format(user.get("first_name"), user.get("last_name"))
     email = user.get("email")
 
+    # send coverage request email copy to current User.
+    recipients.append(email)
```

Comment:
current user should be rather in cc and not actual recipient

Reply:
alright, and one more question— is this change CP-specific? If so, we should consider adding a config option to make it configurable.

Reply:
yep that would make sense :+1: